src/GraphQLParser/Exceptions/GraphQLParserException.cs

```diff
@@ -102,10 +103,4 @@ private static bool HasReplacementCharacter(string str)
     }
 
     private static bool IsReplacementCharacter(char code) => code < 0x0020 && code != 0x0009 && code != 0x000A && code != 0x000D;
-
-    private static string GetUnicodeRepresentation(char code) => code switch
-    {
-        '\0' => "\\u0000",
```

Comment:
I don't know why does this line here. `((int)'\0').ToString("D4")` gives `"\\u0000"` as well.

Reply:
Probably just an optimization for null characters to prevent an allocation or something.

Reply:
Looks strange among all other string allocations above.